services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -693,6 +693,12 @@ class FlowCrudSpec extends HealthCheckSpecification {
         northbound.deleteLink(islUtils.toLinkParameters(newIsl))
         Wrappers.wait(WAIT_OFFSET) { assert !islUtils.getIslInfo(newIsl).isPresent() }
         database.resetCosts()
+        [isl.srcSwitch.dpId, isl.dstSwitch.dpId].each { swId ->
+            with(northbound.validateSwitchRules(swId)) {
+                it.missingRules.empty
+                it.excessRules.empty
+            }
+        }
```

Comment:
What is this? Does not look like a cleanup. If you need to do additional verifications, do them in the 'test' part of the test. We can discuss this, but currently this stands out a bit from our regular style of test